Parsers/ASimNetworkSession/ProductParsers/imNetworkSession.yaml

```diff
@@ -15,6 +15,35 @@ References:
 Description: |
   ASIM Source Agnostic NetworkSession Parser
 ParserName: imNetworkSession
+ParserParams:
+  - Name: starttime
+    Type: datetime
+    Default: datetime(null)
+  - Name: endtime
+    Type: datetime
+    Default: datetime(null)
+  - Name: srcipaddr_has_any_ipv4_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: dstipaddr_has_any_ipv4_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: dstportnumber
+    Type: int
+    Default: int(null)
+  - Name: url_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: httpuseragent_has_any
+    Type: dynamic 
+    Default: dynamic([])
+  - Name: hostname_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: dvcaction
+    Type: dynamic
+    Default: dynamic([])
+
```

Comment:
Don't you need to add parameters to the function calls below?

Reply:
yap!